src/test/regress/sql/distributed_functions.sql

```diff
@@ -705,3 +705,6 @@ DROP SCHEMA function_tests2 CASCADE;
 
 DROP USER functionuser;
 SELECT run_command_on_workers($$DROP USER functionuser$$);
+
```

Comment:
As we stop syncing metadata after adding functions and procedures, we should remove calls like `SELECT wait_until_metadata_sync(30000)` or `select bool_and(hasmetadata) from pg_dist_node WHERE isactive AND  noderole = 'primary';` from distributed_function and distributed_procedure tests.

Reply:
yes, makes sense, let me remove those